### PR TITLE
PHP 8: clean up `Panel` package

### PR DESCRIPTION
**Pull request by distantnative**

### CODE REVIEW

File: src/Panel/Panel.php [hunk 1]
```diff
@@ -289,11 +262,8 @@ public static function response($result, array $options = [])
 
 	/**
 	 * Router for the Panel views
-	 *
-	 * @param string $path
-	 * @return \Kirby\Http\Response|false
 	 */
-	public static function router(string $path = null)
+	public static function router(string|null $path = null): Response|null
```
Comment (afbora) [inline src/Panel/Panel.php:266]: Since `Router::execute` returns `mixed`, shouldn't it return `mixed`?

Comment (distantnative) [reply src/Panel/Panel.php:266]: Doesn't `Router::execute` return what the callback closure returns? Which would be here `static::response()`.

Comment (afbora) [reply src/Panel/Panel.php:266]: Of course, you're right 👍  I missed the closure.

**Review by afbora — APPROVED** [review]
LGTM 👍

**Pull request merged**